internal/crossdock/main_test.go

```diff
@@ -168,7 +169,7 @@ func TestCrossdock(t *testing.T) {
 				"server_oneway": "127.0.0.1",
 			},
 			axes: axes{
-				"transport_oneway": []string{"http"},
+				"transport_oneway": getTransportOnewayAxes(),
```

Comment:
Good enough for now. Might be worth clawing this back to be filtered through Crossdock environment variables.